tests/test_query_across_tables.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+import synapseclient as sc
+from synapsebridgehelpers import query_across_tables
+
+syn = sc.login()
+
+TESTING_TABLE = "syn18503475"
+TESTING_TABLE_2 = "syn18503678"
+
```

Comment:
How do you handle access to these.  Safer options are:
1)  to create these in a startup hook for your test or 
2) test these functions using a unit tests with local data.  
3) Perform integration tests with mocked Synapse functionality.

Reply:
note to self: TODO